infra/dev/rails/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:8.11.3-slim as node_build
+
+COPY yarn.lock .
+
+RUN yarn
+
+FROM ruby:2.5.1 as ruby_build
```

Comment:
It would be nice to build/install ruby based on the .ruby-version file so that it's not hard coded in two places. This feels more like a future task though.

Reply:
Might be possible using build args 👍